src/nav/mobile-monitoring.yml

```
@@ -85,11 +73,12 @@ pages:
                 path: /docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps
               - title: iOS device obfuscation
                 path: /docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-device-id-obfuscation
+
           - title: tvOS
             pages:
               - title: tvOS compatibility and requirements
                 path: /docs/mobile-monitoring/new-relic-mobile-ios/tvos/new-relic-tvos-compatibility-requirements
-          - title: Troubleshoot iOS problems
+          - title: Troubleshoot
```

```suggestion
          - title: Troubleshooting
```

To match "Configuration" as a gerund